lib/stb_image.ex

```diff
@@ -231,13 +231,13 @@ defmodule StbImage do
 
   The supported formats are #{@encoding_formats_string}.
 
-  The format is determined from the file extension if possible,
-  you can also pass it explicitly via the `:format` option.
+  The format is determined from the file extension, if possible.
+  You can also pass it explicitly via the `:format` option.
 
   Returns `:ok` on success and `{:error, reason}` otherwise.
 
-  Make sure the directory you intend to write the file to exists,
-  otherwise an error is returned.
+  Make sure the directory you intend to write the file to exists.
+  Otherwise, an error occurs.
```

Comment:
It's best to be explicit if the error is raised or returned :)

```suggestion
  Otherwise, an error is returned.
```